_component/tabs/index.md

```diff
@@ -18,14 +18,22 @@ iframe_height: medium
 
 {% include partials/html-component.md %}
 
-<h3>SCSS <span class="link"><a href="scss/component.scss" target="_blank">Download SCSS</a></span></h3>
+<h3>PostCSS <span class="link"><a href="scss/component.scss" target="_blank">Download PostCSS</a></span></h3>
 
 {% include partials/scss-component.md %}
```

Comment:
@colorful-tones @garand this is outputting the PostCSS, the recommendation is to import from the npm package, so let's update this to reflect that and if we're using a partial let's not call it `scss-component`

Reply:
This has been addressed. Thanks!